rotkehlchen/tests/api/test_adex.py

```diff
@@ -110,12 +105,8 @@ def test_get_events(
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
 
     # Set module premium is required for calling `get_balances()`
-    premium = None
-    if start_with_valid_premium:
-        premium = Premium(rotki_premium_credentials)
-
     adex = rotki.chain_manager.get_module('adex')
-    adex.premium = premium
+    adex.premium = rotki.chain_manager.premium
```

Comment:
No no no this is not what I said. What part of [what I said](https://github.com/rotki/rotki/pull/3391#discussion_r698344148) was not clear? I even point to you where in the code to change this.

Sure, you should remove it from all these places you removed  it BUT you should instead move the instantiation to the fixture creation. At the exact point my comment links you to. You can loop through the modules of the chain manager there and pass the premium object.